core/templates/services/question-validation.service.ts

```diff
@@ -34,6 +34,10 @@ export class QuestionValidationService {
   isQuestionValid(
       question: Question,
       misconceptionsBySkill: MisconceptionSkillMap): boolean {
+    if (question === undefined || question === null) {
```

Comment:
The question is only of type Question but you are checking for undefined and null. Is this expected. Or should the type definition change?

Reply:
type definition updated